modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -43,7 +61,7 @@ struct Text
     cv::Scalar  color;              //!< The text color
     int         thick;              //!< The thickness of the lines used to draw a text
     int         lt;                 //!< The line type. See #LineTypes
-    bool        bottom_left_origin; //!< When true, the image data origin is at the bottom-left corner. Otherwise, it is at the top-left corner
+    BackendT    backend;            //!< Backend which create mask with text (OpenCV/FreeType)
```

Comment:
May be we could make it a CMake OPTION?

Reply:
For sure it needs to be discussed, maybe our tomorrow's forum is good for that

Reply:
Agree with CMake option

Reply:
Conditional compilation should be avoided in public headers.

Reply:
Solved